### DISCUSSION TITLE
Liquid question

### DISCUSSION THREAD

**Discussion by intellimedhu** [OrchardCMS/OrchardCore#9232] [Q&A]
How can I access Model properties in liquid?

My model is a simple class with a ContentItem property.
If I write `{{ Model | json }}` I can see a valid json, but when referencing properties I have nothing:

`{{ Model | json }}` result:
```
{"ContentItem":{"ContentItemId":"4yk9fzt2yqh3m4402btppc2j5v","ContentItemVersionId":"4zcp9z9ck2xnaxq232v567s0m6", ...
```
---
but `{{ Model.ContentItem | json }}` results ` null `

**Comment by jtkech** [comment]
@intellimedhu 

In your module startup you need to register an access strategy for your model

            services.Configure<TemplateOptions>(options =>
            {
                options.MemberAccessStrategy.Register<YourModelType>();
            });
